### ISSUE TITLE
Datasheet for developers cable

### ISSUE THREAD

**Issue by bgaster** [sensel/sensel-api-arduino#2]
Is there a datasheet for using the Sensel morph with the developers cable?

I'm particularly interested in expected power for each of the lines, i.e. is it expected that TX/RX are running at 5V, as USB requires a range between 4.75V and 5.25V, and of course if there are any hard timings for the sending and receiving data.

**Comment by nyboer** [comment]
Yes - it is expected 5V. You can use a logic level converter like this for 3.3V systems
https://www.sparkfun.com/products/12009

**Comment by bgaster** [comment]
Thanks, I've got this all working now with SM32, there was a small issue with the connection that was causing the line to drop below 3.3v. I've now resolved this. 

Is there documentation for registers read/writes for how much data they return/expect?
